tests/fsharp/FSharp.Tests.FSharpSuite.DrivingCoreCLR/Program.fs

```diff
@@ -0,0 +1,12 @@
+module Program
```

Comment:
Hmmm we don't copyright the test files in the Cambridge suite. or at least not consistently.  I wonder why.